platform/macos/src/MGLMapView.mm

```diff
@@ -252,7 +252,7 @@ + (NSArray *)restorableStateKeyPaths {
     return @[@"camera", @"debugMask"];
 }
 
-- (void)commonInit {
+- (void)commonInit:(nullable NSString*)fontFamily {
```

Comment:
Per https://github.com/mapbox/mapbox-gl-native/pull/10522/#issuecomment-347621074, I think NSFontDescriptor/UIFontDescriptor is the more forward-compatible datatype for this option. It’s important to consider forward compatibility for iOS especially, because changing the type of a property or parameter will require a semver-major release.

If we aren’t ready to support the many options in NSFontDescriptor/UIFontDescriptor, we can specify in the documentation that only the font family option is supported for now.

Reply:
Does @akitchen's change from #10612 dodge the forward-compatibility issue well enough in your mind?

Reply:
Yes, it’s good enough for now. In the future, if we open up more options, then we can introduce formal APIs for them without worrying about backwards compatibility.